Makefile.toml

```diff
@@ -4,6 +4,9 @@ FEATURES = "ecc608"
 BUILD_COMMAND = "cross"
 TAR = { source = "${CARGO_MAKE_RUST_TARGET_OS}", default_value = "linux", mapping = {"macos" = "gtar", "linux" = "tar" } }
 
+[env.x86_64-unknown-linux-gnu]
```

Comment:
Rename to `x86_64-ecc-debian-gnu` to match the tpm one?

Reply:
Ah yes.. so then `x86_64-unknonw-debian`. I tripped more over the use of linux than the tpm/unknown part

Reply:
@madninja ok - fixed and rebased